src/server/internalServer.cpp

```diff
@@ -288,13 +293,14 @@ SearchInfo InternalServer::getSearchInfo(const RequestContext& request, const st
     throw Error(nonParameterizedMessage("no-query"));
   }
 
-  return SearchInfo(pattern, geoQuery, bookIds);
+  return SearchInfo(pattern, geoQuery, bookIds.second, bookIds.first);
 }
 
-SearchInfo::SearchInfo(const std::string& pattern, GeoQuery geoQuery, const Library::BookIdSet& bookIds)
+SearchInfo::SearchInfo(const std::string& pattern, GeoQuery geoQuery, const Library::BookIdSet& bookIds, const std::string& searchQuery)
```

Comment:
`searchQuery` is misleading in this context. `bookFilter` or `bookFilterQuery` would describe the meaning of this variable better.

Reply:
Indeed. Done in last fixup commit